decentraland/unity-renderer
# Chore: move implicitly referenced assets out of Resources folder

## What does this PR change?
Relates #4369 

- moved assets that wasn't directly referenced by `Resources.Load` out of the Resources folder

### Affected systems:
- AvatarEditorHUD
- UIScreenSpace
- BackpackEditorV2
- FriendsHUD
- NotificationBadge
- Pasport
- PlayerInfoCardHUD
- QuestPanelHUD
- SettingsPanelHUD
- TaskbarHUD
- MapRenderer
- PlayerName
- GreenBlockers

## How to test the changes?

1. Launch the explorer
2. Check that mentioned above systems are working as in prod

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
